luigi/worker.py

```diff
@@ -786,7 +782,7 @@ def _run_task(self, task_id):
 
         self._running_tasks[task_id] = p
 
-        if self.worker_processes > 1:
+        if p.random_seed:
```

Comment:
I like PR's to be focused as well. But if you decide to only minimally touch hard-to-understand code, it will remain hard-to-understand for the reviewers too. I would appreciate it if you could change the name of this local variable at this one place. Not strictly necessary though (as with all my suggestions :)).